app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class QuotesController < ApplicationController
+ def search
```

Comment:
see my comment in your `QuoteFacade`. If you refactor that to then create a PORO, your controller would then potentially look like:
```
@search_phrase = params[:query]
    @total_found, @quotes_list = QuotesFacade.quotes(@search_phrase)
```